### PR TITLE
Standardize inbound MM Resolved Model IDs to vModelIDs

### PR DESCRIPTION
**Pull request by RobGeada**
Many thanks for submitting your Pull Request :heart:!

Please make sure that your PR meets the following requirements:

- [ ] You have read the [contributors guide](https://github.com/trustyai-explainability/trustyai-explainability/blob/main/CONTRIBUTING.md)
- [ ] Pull Request title is properly formatted: `FAI-XYZ Subject`
- [ ] Pull Request title contains the target branch if not targeting main: `[0.3.x] FAI-XYZ Subject`
- [ ] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains link to any dependent or related Pull Request
- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket

### CODE REVIEW

**Review by tteofili — APPROVED** [review]

File: explainability-service/src/main/java/org/kie/trustyai/service/data/utils/ModelMeshInferencePayloadReconciler.java [hunk 1]
```diff
@@ -36,6 +36,14 @@ public class ModelMeshInferencePayloadReconciler extends InferencePayloadReconci
     @Inject
     Instance<DataSource> datasource;
 
+    private String standardizeModelId(String inboundModelId) {
+        if (inboundModelId != null && inboundModelId.contains("__isvc")) {
```
Comment (ruivieira) [inline explainability-service/src/main/java/org/kie/trustyai/service/data/utils/ModelMeshInferencePayloadReconciler.java]: @RobGeada would it be ok to make this a constant?

File: explainability-service/src/main/java/org/kie/trustyai/service/data/utils/ModelMeshInferencePayloadReconciler.java [hunk 2]
```diff
@@ -36,6 +36,14 @@ public class ModelMeshInferencePayloadReconciler extends InferencePayloadReconci
...
+            return inboundModelId.split("__isvc")[0];
```
Comment (ruivieira) [inline explainability-service/src/main/java/org/kie/trustyai/service/data/utils/ModelMeshInferencePayloadReconciler.java]: @RobGeada what about the case of multiple occurences? i.e. a model named `my__isvc_model__isvc-xxxxx"?

Perhaps something like

```
int index = inboundModelId.lastIndexOf("__isvc");
return inboundModelId.substring(0, index);
```

File: explainability-service/src/main/java/org/kie/trustyai/service/data/utils/ModelMeshInferencePayloadReconciler.java [hunk 3]
```diff
@@ -36,6 +36,14 @@ public class ModelMeshInferencePayloadReconciler extends InferencePayloadReconci
...
+        } else {
+            return inboundModelId;
+        }
+    }
+
```
Comment (ruivieira) [inline explainability-service/src/main/java/org/kie/trustyai/service/data/utils/ModelMeshInferencePayloadReconciler.java:49]: @RobGeada Would it be ok to add a few simple unit tests? e.g. model name with
- no `__isvc`
- standard name with `__isvc`
- multiple `__isvc`

**Review by ruivieira — CHANGES_REQUESTED** [review]

**Review by ruivieira — APPROVED** [review]
@RobGeada LGTM.

There seem to be just some errors with formatting.

**Pull request merged**